### ISSUE TITLE
how to create time series raster layers with self-produced rasters

### ISSUE THREAD

**Issue by myidream** [loicdtx/bfastSpatial#80]
Hi guys! I am a new user of “bfastSpatial” R package to detect the breakpoint of vegetation gross primary productivity (GPP) of China during the past 15 years based the monthly spatial GPP data that we produced using VPM model with MODIS images.  I meet the problem in the step to create time series GPP data using the timeStack() command. I am not sure whether the “bfastSpatial” package can be applied in raster layers that not from Landsat or MODIS. Any reponse would be helpful.

**Comment by loicdtx** [comment]
That's right, `timeStack()` is specific to Landsat and MODIS land data so that it won't work on data that follow other naming conventions.
The good thing is that `timeStack()` is really just a thin wrapper around `raster::stack()` + `raster::setZ()`, so that it should be easy to adapt or bypass. See the code [here](https://github.com/loicdtx/bfastSpatial/blob/develop/R/timeStackMODIS.R).
The key in your case will be to get a vector of dates corresponding to each layer that will compose your stack.
